components/tabs/tabs.js

```diff
@@ -713,6 +733,37 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(SkeletonMixin(LitElement))
 	}
 
 	async _handleTabSelected(e) {
+		if (this._defaultSlotBehavior) {
+			this._handleTabSelectedDefaultSlotBehavior(e);
+			return;
+		}
+
+		e.stopPropagation();
```

Comment:
If we want `d2l-tab-selected` to be the new `d2l-tab-panel-selected` then this should be removed.